docs/manual.xml.head

```diff
@@ -15958,6 +15958,20 @@ virtual-mailboxes "My INBOX" "notmuch://?query=tag:inbox"
                   <literal>year</literal>
                 </entry>
               </row>
+              <row>
+                <entry><literal>nm_query_window_additional_terms</literal></entry>
```

Comment:
The name feels long, but is certainly descriptive.  Do you like `nm_query_window_extras` any better?  Shorter, but not as precise.

Reply:
> The name feels long

Agreed.

>  Do you like `nm_query_window_extras` any better?

I think that reduces the descriptiveness too much. Maybe something like `nm_query_window_or_terms` instead? It hints more towards an `OR` statement in the window. I think users will need to look at the manual regardless for more details.